package.json

```diff
@@ -671,7 +671,7 @@
     "ts-node": "^10.9.2",
     "tsx": "^4.19.2",
     "ttest": "^2.1.1",
-    "typescript": "~5.4.5",
+    "typescript": "~5.7.2",
```

Comment:
For some reason this was required to get the override in the `types` package to work with `.ts` (even though the older version worked fine with `.mts` previously).